### PR TITLE
Contact Us Page

### PR DESCRIPTION
**Pull request by aleckslu**
- Contact page's title and text should be server side rendered
- contact form should be client side rendered
- filled out forms will be sent to pomoparty support email
- form currently doesn't provide instant feedback & only has default requirements for form submission

### CODE REVIEW

**Review by sn163 — CHANGES_REQUESTED** [review]

File: src/actions/handleWeb3.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+"use server";
+
+export async function submitWeb3Form(formData: FormData) {
+  console.log("this event in submit web3form:  ", formData);
```
Comment (sn163) [inline src/actions/handleWeb3.tsx:4]: Remove test console.logs

**Review by sn163 — CHANGES_REQUESTED** [review]

**Comment by sn163** [comment]
The action folder needs to have the `_` prefix too

File: src/actions/handleWeb3.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
+"use server";
```
[the remark this answers is not in the document]
Comment (aleckslu) [reply src/actions/handleWeb3.tsx:1]: for server actions, u gotta include the "use server".  thatll also allow you to import those actions and use it on client components from my understanding

File: src/actions/handleWeb3.tsx (hunk 1 above)
Comment (aleckslu) [reply src/actions/handleWeb3.tsx:4]: Sorry, forgot to remove that

**Comment by aleckslu** [comment]
> The action folder needs to have the `_` prefix too. Other than that it looks good so far. Form styling and positioning needs work but we can do that in future updates

I was wondering about that too actually. But every where on docs, other projects, and articles I've seen the actions folder is named without the `_`.  For instance, all the examples on the docs ( [link](https://nextjs.org/docs/app/building-your-application/data-fetching/server-actions-and-mutations#pending-states) ) all the `actions/` directories are named without the `_` even when they're directly under `app/`

I'll still change the name to `_actions/` and push it up.

File: src/actions/handleWeb3.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (sn163) [reply src/actions/handleWeb3.tsx]: Ah got it, makes sense. Good catch, looks good

**Review by sn163 — APPROVED** [review]

**Pull request merged**